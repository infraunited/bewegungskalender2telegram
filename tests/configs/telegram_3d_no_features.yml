output:
  type: 'telegram'
  stdout: true
caldav:
  url: 'http://127.0.0.1:18476'
  calendars:
    - url: '/d/5cbae8f0-5881-c7a2-4cbb-c5c1ba05bc25/'
    - url: '/d/701e875a-44ca-97df-0140-bbb36dc26a9b/'
  username: 'x'
  password: 'x'
  offset_days: 0
  number_days: 3
  current_day_override: "2023-01-01"
format:
  time_locale: "de_DE.utf8"
  timezone: "Europe/Berlin"
  show_calendar_name: false
  show_location: false
  show_description: false
  cutoff_description: 0
  header_txt: ""
  footer_txt: ""
