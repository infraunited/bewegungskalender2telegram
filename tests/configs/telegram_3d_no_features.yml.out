ðŸ“… *__Sonntag__ 2023\-01\-01*

*Multi\-All\-Day*

*Single All\-Day*

__08:30__ *Long timed event*

__09:00__ *Single Event*


ðŸ“… *__Montag__ 2023\-01\-02*

*Multi\-All\-Day*

*Long timed event*

__10:00__ *\-\>test \[s@\_\+\}*


ðŸ“… *__Dienstag__ 2023\-01\-03*

*?*

*Multi\-All\-Day*

__00:00__ *Long timed event*

__00:00__ *Recurring twice weekly*

__00:00__ *?*

__07:30__ *An event without time*